asf_public_discourse_home_decarbonisation/analysis/buildhub/sentence_topic_and_sentiment_analysis_for_bh.py

```diff
@@ -0,0 +1,332 @@
+            "86_inverter_driven_inverters_drive",
+        ],
+        "Noise": [
+            "4_noise_sound_noisy_quiet",
+        ],
+        "Miscellaneous Components": [
+            "49_flow_rate_rates_meter",
+            "50_wiring_cable_wires_wire",
+            "65_zones_zone_zoning_single",
+            "121_phase_single_3phase_transformer",
+            "127_blinds_external_windows_velux",
+            "135_warranty_year_manufacturer_item",
+            "137_trvs_trv_open_rads",
+            "143_batteries_battery_storage_life",
+        ],
+    },
+    "Installation": [
+        "10_installer_installers_install_installation",
+        "25_plumber_plumbers_plumbing_electrician",
+        "94_vaillant_daikin_installer_schematics",
+        "118_manual_manuals_instructions_copy",
+    ],
+    "Temperature and Control": [
+        "13_flow_temp_temperature_temps",
+        "36_thermostat_thermostats_room_control",
+        "55_controller_controls_control_controllers",
+        "79_degrees_temperature_temp_set",
+        "91_settings_setting_parameters_change",
+        "147_ftc6_ftc_ftc5_controller",
+    ],
+    "Certification and Incentives": [
+        "21_certification_microgeneration_scheme_certificate",
+        "51_grant_bus_grants_5k",
+        "96_epc_incentive_renewable_rating",
+        "136_certification_microgeneration_scheme_certified",
+    ],
+    "Energy and Efficiency": [
+        "52_carbon_co2_emissions_fossil",
+        "62_octopus_tariff_cosy_agile",
+        "115_efficiency_efficient_inefficient_penalty",
+        "108_tariff_tariffs_economy_tou",
+        "126_energy_consumption_efficiency_rating",
+    ],
+    "Design and Planning": [
+        "67_planning_lpa_pd_permission",
+        "69_size_sizing_oversizing_fit",
+        "111_design_designers_designer_professional",
+    ],
+    "Weather and Climate": [
+        "32_winter_weather_cold_rain",
+        "85_house_warm_cold_days",
+        "110_climate_change_global_scientists",
+    ],
+    "Calculation and Data": [
+        "46_loss_calculations_calcs_spreadsheet",
+        "112_spreadsheet_excel_chart_data",
+        "56_numbers_figures_maths_calculations",
+    ],
+    "Water Systems": [
+        "57_tank_tanks_loft_litre",
+        "64_cylinder_domestic_hot_water",
+        "75_legionella_cycle_legionnaires_bacteria",
+        "81_domestic_hot_kwh_water",
+        "107_tank_temperature_temp_degrees",
+        "109_cylinder_reheat_unvented_cylinders",
+        "129_immersion_immersions_3kw_solic",
+    ],
+    "Miscellaneous": [
+        "2_thread_post_question_forum",
+        "63_build_self_builders_builder",
+        "66_garden_kitchen_room_bedroom",
+        "104_loops_loop_100m_loopcad",
+        "105_2023by_2022by_editeddecember_2021by",
+        "119_trees_wood_tree_timber",
+        "128_200mm_150mm_100mm_spacing",
+        "130_photos_pics_photo_picture",
+        "132_problem_issue_issues_problems",
+        "124_e7_e10_domestic_boost",
```

Comment:
e7 and e10 are tariffs, so potentially if you create a group for tariffs, this could go there.